README.rst

```diff
@@ -0,0 +1,224 @@
+
+.. code-block:: python
+
+    burst_kwargs = {'amplitude_fraction_threshold': .2,
+                    'amplitude_consistency_threshold': .5,
+                    'period_consistency_threshold': .5,
+                    'monotonicity_threshold': .8,
+                    'N_cycles_min': 3}
+
+    narrowband_kwargs = {'N_seconds': .5}
+
+    df = compute_features(signal, Fs, f_range,
+                          center_extrema='T',
+                          burst_detection_method='cycles',
+                          burst_detection_kwargs=burst_kwargs,
+                          find_extrema_kwargs={'filter_kwargs': narrowband_kwargs},
+                          hilbert_increase_N=True)
+
+
+- **center_extrema** determines how the cycles are segmented. 'T' indicates the center extrema is \
+  a trough, so cycles are segmented peak-to-peak.
+- **burst_detection_method** selects which method for burst detection is used. The 'cycles' option \
+  uses features of adjacent cycles in order to detect bursts (e.g. period consistency, see next \
+  item). The 'amp' option uses an amplitude threshold to determine the cycles that are part of an \
+  oscillatory burst.
+- **burst_detection_kwargs** set the keyword arguments for the burst detection function. For the \
+  ``cycles`` method, there are 5 keyword arguments (see the end of the \
+  `algorithm tutorial <https://bycycle-tools.github.io/bycycle/auto_tutorials/plot_2_bycycle_algorithm.html#sphx-glr-auto-tutorials-plot-2-bycycle-algorithm-py>`_ \
+  for advice on choosing these parameters).
+- **find_extrema_kwargs** set the keyword arguments for the function used to localize peaks and \
+  troughs. Most notably, you can change the duration of the bandpass filter (``N_seconds``) used \
+  during extrema localization (see section 1 of the \
+  `algorithm tutorial <https://bycycle-tools.github.io/bycycle/auto_tutorials/plot_2_bycycle_algorithm.html#sphx-glr-auto-tutorials-plot-2-bycycle-algorithm-py>`_).
+- **hilbert_increase_N** is a boolean indicator of whether or not to zeropad the signal to bypass \
+  complications that ``scipy.signal.hilbert()`` has with some long signal durations. Try setting \
+  this parameter to `True` if this function is taking a long time to run. Note the Hilbert \
+  Transform is used to compute the `band_amp` feature of each cycle, which is the average analytic \
+  amplitude of the frequency of interest in that cycle. This is complementary to the `volt_amp` \
+  measure, and may be desired for some burst detection applications.
+
+Output
+------
+
+The output of ``bycycle`` is a ``pandas.DataFrame``, a table like the one shown below (with many
+columns, so it is split into two images).
+
+Each row of this table corresponds to an individuals segment of the signal, or a putative cycle of
+the rhythm of interest.
+
+.. image:: https://github.com/bycycle-tools/bycycle/raw/master/img/cycledf_1.png
```

Comment:
This looks great! Since these images are for the docs, can you move the `img` folder into the `doc` folder, and then update these links?

Reply:
I moved the img folder inside of doc and update these links for when this PR is merged into master.